sub-scripts/commons.sh

```diff
@@ -31,6 +31,24 @@ function plural {
     fi
 }
 
+function retry_command {
```

Comment:
@Father1993 maybe `retry_it()` would be a more suitable name, taking into account all other names in this file?